### PR TITLE
Unified support page

### PR DESCRIPTION
**Pull request by pk-pranshu**
Created a support page for Fluid framework customers. This is the place where they will get all information related to support and help going forward. Also, included SPE and AFR support page links.

Please Note: This is just the first version to get us started. Will improve, prettify and add more links in the next iteration.

### CODE REVIEW

**Comment by borisf-msft** [comment]
I would rephrase _"If you have any questions or need assistance with Fluid Framework, please reach out to our support team. We are here to help!"_. This implies an actual "support team" (aka Microsoft Support) for FF.

... "please reach out to us on Github!" may be more accurate and set more accurate expectations.

**Comment by kashms** [comment]
Content looks good to me.

I don't have a strong opinion on this, but should this be in a Q&A format or just a list of support resources? Q&A format breaks up the big picture support "flow-chart". If issues >go here, if questions > go here, for AFR > go here etc.

**Comment by pk-pranshu** [comment]
I am okay changing the format in the next iteration. I feel for now I want to start with this only to have our first landing. Would love to have logos and graphics on this page in the next iteration and remove the QnA format

**Comment by pk-pranshu** [comment]
> I would rephrase _"If you have any questions or need assistance with Fluid Framework, please reach out to our support team. We are here to help!"_. This implies an actual "support team" (aka Microsoft Support) for FF.
> 
> ... "please reach out to us on Github!" may be more accurate and set more accurate expectations.
> 
> @pk-pranshu

Updated

**Review by kashms — APPROVED** [review]
Sounds good, let's get this out! 🚢

**Review by borisf-msft — APPROVED** [review]
